vega/static/vega-lite.html

```diff
@@ -1 +1,11 @@
-<div class="vega-embed" data-type="vega-lite">{spec}</div>
+<div class="vega-embed" id="{id}"></div>
+
+<style>
```

Comment:
Yeah, seemed a tad simpler, but I don't mind going back...

On Tue, May 17, 2016 at 3:41 PM, Dominik Moritz <notifications@github.com>
wrote:

> In vega/static/vega-lite.html
> <https://github.com/vega/ipyvega/pull/32#discussion_r63617094>:
>
> > @@ -1 +1,11 @@
> > -<div class="vega-embed" data-type="vega-lite">{spec}</div>
> > +<div class="vega-embed" id="{id}"></div>
> > +
> > +<style>
>
> Did you do this so that we don't need the js file for styling?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/vega/ipyvega/pull/32/files/8fe2e0feea95138022e230298c9c4f4efbf0252e#r63617094>
>



-- 
Brian E. Granger
Associate Professor of Physics and Data Science
Cal Poly State University, San Luis Obispo
@ellisonbg on Twitter and GitHub
bgranger@calpoly.edu and ellisonbg@gmail.com